src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -86,10 +86,11 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
 
   const joinTable = useCallback(
     (table) => {
-      joinHuddle(`${venue.id}-${table}`);
+      // @debt TOOD Fix this
```

Comment:
There's a `jazzbar` branch that deals with this

Reply:
Which one gets to be merged first and into which base?